client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -136,14 +136,14 @@ class Upload extends React.Component {
 		debug( 'Error', { error } );
 
 		const errorCauses = {
-			exists: translate( 'Upload problem: Theme already installed on site.' ),
-			already_installed: translate( 'Upload problem: Theme already installed on site.' ),
-			'too large': translate( 'Upload problem: Theme zip must be under 10MB.' ),
-			incompatible: translate( 'Upload problem: Incompatible theme.' ),
-			unsupported_mime_type: translate( 'Upload problem: Not a valid zip file' ),
+			exists: translate( 'Install problem: Theme already installed on site.' ),
+			already_installed: translate( 'Install problem: Theme already installed on site.' ),
+			'too large': translate( 'Install problem: Theme zip must be under 10MB.' ),
+			incompatible: translate( 'Install problem: Incompatible theme.' ),
+			unsupported_mime_type: translate( 'Install problem: Not a valid zip file' ),
 			initiate_failure: translate(
-				'Upload problem: Theme may not be valid. Check that your zip file contains only the theme ' +
-					'you are trying to upload.'
+				'Install problem: Theme may not be valid. Check that your zip file contains only the theme ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Upload problem: Theme may not be valid. Check that your zip file contains only the theme you are trying to upload.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).